openverse_catalog/dags/providers/provider_csv_load_scripts/inaturalist/create_schema.sql

```diff
@@ -0,0 +1,4 @@
+CREATE SCHEMA IF NOT EXISTS inaturalist;
```

Comment:
If the schema does exist, is that an error condition or is it safe to assume that we can move forward? Will downstream errors make it clear when this isn't working as expected?

Reply:
I just wrote a whole long rambling thing, but I think at the end of the day, yes, if the schema exists when airflow is kicking off the dag, something is not right. And, as I think about the kinds of things that might be wrong (mostly having to do with very long run times and stopping in the middle), I'm not sure that downstream errors would catch them. Take it out?

Reply:
I really don't feel confident enough about how airflow will handle retries to have a strong opinion about what's best here.

Reply:
I guess maybe we could take out the `IF NOT EXISTS` bit and let an error come up (I'm assuming PG raises an error if you try to create something that already exists? :thinking:)

Reply:
It will attempt to raise an error, but in this case I think it's fine to leave the `IF NOT EXISTS` clause in. I could imagine a scenario where some cleanup step doesn't perform correctly or we have to retry a step without the cleanup steps having run yet - such a situation actually happened with this DAG which is why we added #634. Having the `IF NOT EXISTS` here makes it a bit more robust to failures and prevents us having to manually go into the database to drop the schema if we want to rerun certain steps. Each of the table creation steps perform a `DROP TABLE IF EXISTS` command at the beginning ([example](https://github.com/WordPress/openverse-catalog/pull/549/files/6282c174f2af737119d803a3d8fe0c5530ea215c#diff-09f2f62474f7850278140b7385d47106b842c6289e05485bbe655dbd80d204a9R18)), so even if the schema did exist whatever data was present would get cleared out before new data was inserted.

Reply:
> Each of the table creation steps perform a DROP TABLE IF EXISTS command at the beginning

This makes perfect sense, thankss for the explanation.

Reply:
Thanks so much @AetherUnbound. That's a much clearer version of what I was trying to say in my deleted rambling. So I'm going to leave this as is.